extension/scripts/content/city/ttCity.js

```diff
@@ -0,0 +1,189 @@
+"use strict";
+
+(async () => {
+    await loadDatabase();
+    if (!settings.pages.city.highlight && !settings.pages.city.showValue) return;
+    
+    console.log("TT - City: Starting...");
+    await requireMapLoaded();
+    run();
+    listenToSettings();
```

Comment:
I think having this as a separate function is overkill and inconsistent with our other content scripts.